tools/code_format/check_format.py

```diff
@@ -417,16 +419,28 @@ def reportError(message):
       # If we see a version marker we are now in the section for the current release.
       in_current_release = True
 
-    # Do basic alphabetization checks of the first word on the line and the
-    # first word after the :
+    # make sure flags are surrounded by ``s
+    flag_match = RELOADABLE_FLAG_REGEX.match(line)
+    if flag_match:
+      if not flag_match.groups()[0].startswith('`'):
+        reportError("Flag `%s` should be enclosed in back ticks" % flag_match.groups()[1])
+
     if line.startswith("*"):
+      if prior_line and not prior_line.startswith('=') and not prior_line.endswith('.'):
+        if prior_line.endswith('>`'):
+          break
+          # We don't punctuation-check sentences ending in :ref yet.
```

Comment:
Is this because there are existing ones (eg in 1.13.0) that we don't want to overwrite history on?

Reply:
nah, it's because I tried about 5 times to sort out how to make sure that the data in the :ref ended in . and decided I was going to just fix the common case - I can only justify so much time on fix_format scripts :-P